content/events/2022-05-papercuts/index.md

```diff
@@ -0,0 +1,83 @@
+* Make reducing paper cuts fun.
+
+This will be a 24h event, spanning all time zones with our worldwide community. We'll start at 09:00 Melbourne time, and end at 16:00 Portland time.
+
+## Interested?
+
+On Papercuts Day, please __join us on [Gitter](https://gitter.im/galaxyproject/Lobby)__ (all day). If you don't have a GitHub or Gitter (or Elements/Matrix) account yet, then set them up (in that order).
+
+
+# Before the event
+
+If you are a Galaxy user or admin, you can create, or tag existing GitHub issues for any irritating, but likely easily fixed bugs (make a guess). These can be in any part of Galaxy, not just the core engine, and includes training, documentation, and other software.
+
+### Resources
+
+If you have time, and want to learn more about contributing you are encouraged to review:
+
+* [A collection of short videos](https://www.youtube.com/playlist?list=PLNFLKDpdM3B8Ro-V0mZboQjj40RimRZ8n), highlighting how to do common useful tasks during Paper Cuts and beyond.
+* [Development in Galaxy tutorials](https://training.galaxyproject.org/training-material/topics/dev/).
+* [Contributing to the Galaxy Training Material tutorials](https://training.galaxyproject.org/training-material/topics/contributing/).
+* [Galaxy on GitHub](https://github.com/galaxyproject) - see what projects are supported and find the one that speaks to you. Pay particular attention to each project's CONTRIBUTING.md files.
+
+# During the event
+
+
+We will be on [Gitter](https://gitter.im/galaxyproject/Lobby) for chat all day long.
+
+There are also two GTN CoFest calls planned for the day.
+
+
+* **[GTN CoFest](https://github.com/galaxyproject/training-material/issues/1712) Call 1: Middle East, Europe, Africa**
+    * 10:00 Central European time.  [See your time](https://www.timeanddate.com/worldclock/fixedtime.html?msg=EMEA+GTN+CoFest+Call&iso=20220519T10&p1=980&am=30).
+    * [Zoom](https://github.com/galaxyproject/training-material/issues/1712)
+* **[GTN CoFest](https://github.com/galaxyproject/training-material/issues/1712) Call 2: Americas**
+    * 11:00 US Eastern time.  [See your time](https://www.timeanddate.com/worldclock/fixedtime.html?msg=Americas+GTN+CoFest+Call&iso=20220519T11&p1=3705&am=30).
+    * [Zoom](https://github.com/galaxyproject/training-material/issues/1712)
+
+## Issues to work on
+
+This papercut day we're going to focus on some **UI-UX** issues for the **New History**. If that's not your specialty, feel free to pick something else OR you can challenge yourself to learn something new!
+
+[You can find the project board and list of issues here!](https://github.com/orgs/galaxyproject/projects/23/views/4)
+
+
+## After the event
+
+*Get those pull requests in! And address anything from the review! And become more involved in the Galaxy code, training, and documentation base!*
+
+
+## Want to host?
+
+If you're a member of a working group and would like to nominate yourselves to host a papercut event, please contact Assunta DeSanto.
```

Comment:
Maybe a preferred way to contact you? Or not needed?

Reply:
If it's just internal, they can find me...